readme.md

```diff
@@ -138,6 +146,24 @@ Returns a [`child_process` instance](https://nodejs.org/api/child_process.html#c
   - is also a `Promise` resolving or rejecting with a [`childProcessResult`](#childProcessResult).
   - exposes the following additional methods and properties.
 
+#### kill([signal], [options])
+
+Same as the original [`child_process.kill()`](https://nodejs.org/api/child_process.html#child_process_subprocess_kill_signal) except: if `signal` is `SIGTERM` (the default value) and the child process is not terminated after 5 seconds, force it by sending `SIGKILL`.
+
+##### options.retry
+
+Type: `boolean`<br>
+Default: `true`
+
+If the first signal does not terminate the child after a specified timout, a `SIGKILL` will be sent to the process.
+
+##### options.retryAfter
+
+Type: `string | number`<br>
+Default: `5000`
```

Comment:
What considerations were given for having 5000 as the default?

Reply:
Originally @zokker13 had it as `2000`. I suggested `5000` because I feel like 2 seconds is not enough to do proper shutdown of some processes. E.g. some servers need to perform some network requests before properly closing.

But that's an edge case, so this sounds good if you want a much smaller default value. Which one do you have in mind?